server/channels/api4/system.go

```diff
@@ -281,7 +281,7 @@ func getAudits(c *Context, w http.ResponseWriter, r *http.Request) {
 	audit.AddEventParameter(auditRec, "audits_per_page", c.Params.LogsPerPage)
 
 	if err := json.NewEncoder(w).Encode(audits); err != nil {
-		c.Logger.Warn("Error while writing response", mlog.Err(err))
+		c.Logger.Warn("Error while writing response")
```

Comment:
Whoops, that's an unrelated change

Reply:
Should I maintain what was there initially?

Reply:
Yes, please.

Why did you remove the  error from the log message in the first place?

Reply:
I omitted it based on the the change that you suggested above. It did not have the `mlog.Err(err)`

Reply:
Oh, my bad. I didn't intent to remove it. Sorry.

Reply:
It's okay, I will apply those changes